### ISSUE TITLE
client-transcribe-streaming 3.2.0 not working in production with NextJS

### ISSUE THREAD

**Issue by ArmandDu** [aws/aws-sdk-js-v3#1918]
**Describe the bug**
When using `@aws-sdk/client-transcribe-streaming@3.2.0` with [NextJS](https://nextjs.org/), we get an "Expected SignatureV4 signer, please check the client constructor" Error.

The error only happens in production but not in dev mode.

using nextjs's dev mode, we can use the transcribe client, but when it's built, then the client stop working.

We've pinpointed the issue to the middleware-sdk-transcribe-streaming [configuration file](https://github.com/aws/aws-sdk-js-v3/blob/master/packages/middleware-sdk-transcribe-streaming/src/configuration.ts).

```javascript
// starting at line 34 to line 37

const validateSigner = (signer: any): signer is BaseSignatureV4 =>
  // We cannot use instanceof here. Because we might import the wrong SignatureV4
  // constructor here as multiple version of packages maybe installed here.
  (signer.constructor.toString() as string).indexOf("SignatureV4") >= 0;
```

The validateSigner expects the signer object's string representation to contain "SignatureV4".

Unfortunately, as we are using it in the front-end, the code is bundled and transformed to make the file size smaller. Removing the "SignatureV4" information.

**SDK version number**
@aws-sdk/client-transcribe-streaming@3.2.0

**Is the issue in the browser/Node.js/ReactNative?**
Browser

**Details of the browser/Node.js/ReactNative version**
Browsers:
    Firefox: 84.0.2

**To Reproduce (observed behavior)**
Minimal Repo : https://github.com/ArmandDu/aws-sdk-js-v3-nextjs-issue

To reproduce the issue, clone the repo, install the dependencies, build the app `yarn build` and start it `yarn start`
The front end page is similar to this example  [https://transcribe-websockets.go-aws.com](https://transcribe-websockets.go-aws.com)

After submitting the Access ID, Secret Key and region, an error should show as an alert popup.

**Expected behavior**
We expect the application to not throw a runtime error when using it in production.

The SDK should be able to detect the correct Signer version and use it for the transcribe streaming command.

**Screenshots**

**Additional context**

**Comment by davidpene** [comment]
Also experiencing this issue with CRA builds.

Are there any updates or workarounds?

**Comment by trivikr** [comment]
> Also experiencing this issue with CRA builds.

The CRA issue is documented at https://github.com/aws/aws-sdk-js-v3/issues/2002

**Comment by davidpene** [comment]
Hey @trivikr has there been any updates on this?

**Comment by trivikr** [comment]
Fixed in https://github.com/aws/aws-sdk-js-v3/pull/2676
It would be released with `v3.27.0` on 08/20
